main/test/ca/uwaterloo/flix/language/phase/TestSafety.scala

```diff
@@ -41,33 +40,56 @@ class TestSafety extends FunSuite with TestUtils {
     val input =
       """
         |pub def f(): #{ A(Int), B(Int), R(Int) } = solve {
-        |    R(x) :- A(x), not B(_y).
+        |    R(x) :- not A(x), not B(x).
         |}
-      """.stripMargin
+    """.stripMargin
     val result = compile(input, DefaultOptions)
     expectError[IllegalNonPositivelyBoundVariable](result)
   }
 
-  test("NonPositivelyBoundVariable.03") {
+  test("NegativelyBoundWildVariable.01") {
     val input =
       """
         |pub def f(): #{ A(Int), B(Int), R(Int) } = solve {
-        |    R(x) :- not A(x), not B(x).
+        |    R(x) :- A(x), not B(_y).
         |}
       """.stripMargin
     val result = compile(input, DefaultOptions)
-    expectError[IllegalNonPositivelyBoundVariable](result)
+    expectError[IllegalNegativelyBoundWildVariable](result)
   }
 
-  test("NonPositivelyBoundVariable.04") {
+
+  test("NegativelyBoundWildVariable.02") {
     val input =
       """
         |pub def f(): #{ A(Int), B(Int), R(Int) } = solve {
         |    R(1) :- not A(_x), not B(_y).
         |}
       """.stripMargin
     val result = compile(input, DefaultOptions)
-    expectError[IllegalNonPositivelyBoundVariable](result)
+    expectError[IllegalNegativelyBoundWildVariable](result)
+  }
+
+  test("NegativelyBoundWildcard.01") {
+    val input =
+      """
+        |pub def f(): #{ A(Int), B(Int) } = solve {
```

Comment:
I would do some tests without `solve` since it is not required for the error to trigger.